apps/src/netsim/netsim.js

```diff
@@ -301,6 +303,15 @@ NetSim.prototype.shouldShowAnyTabs = function () {
 };
 
 /**
+ * @returns {boolean} TRUE if the "resetShard" flag is found in the URL
+ * TODO: This needs to be replaced with real UI and a route that
+ *       only allows section owners and admins to perform a reset.
+ */
+NetSim.prototype.shouldResetShard = function () {
+  return location.search.match(/resetShard/i);
```

Comment:
I suppose this is just a temporary measure, but even so this seems like a pretty wide net